.github/workflows/release.yml

```diff
@@ -30,6 +30,8 @@ jobs:
         run: yarn install --check-files --frozen-lockfile
       - name: release:production
         run: npx projen release:production
+      - name: Save CloudFormation templates
```

Comment:
Hier kopieer je de templates, maar er gebeurt zo te zien niks meer mee daarna? Of ik snap 'm niet helemaal, wat goed mogelijk is.

Reply:
Aah ok. Het kan verder geen kwaad zo te zien.